src/api/interface/index.ts

```diff
@@ -49,6 +42,37 @@ export interface Permission {
   name: string;
 }
 
+export interface TestInterpretationDTO {
+  namePattern: string;
+  theme: string;
+  variables: object;
+}
+
+export interface TestTypeDTO {
+  id: string;
+  name: string;
+  resultsSchema: object;
+  neededPermissionToAddResults: string;
+}
+
+export interface TestDTO {
+  id: string;
+  userId: string;
+  creationTime: Date;
+  testType: TestTypeDTO;
```

Comment:
Love this change.

Reply:
Everybody does :) Alas some times it's necessary